pkg/api/kyverno/v1/policy_types.go

```diff
@@ -54,6 +54,12 @@ type Spec struct {
 	// uses variables that are only available in the admission review request (e.g. user name).
 	// +optional
 	Background *bool `json:"background,omitempty" yaml:"background,omitempty"`
+
+	// ShemaValidation controls skipping policy validation checks.
+	// Optional. Default value is "true". The value must be set to "false" if policy should
+	// skip validation checks.
```

Comment:
Suggest change to:

>// ShemaValidation skips policy validation checks.
// Optional. The default value is set to "true", it must be set to "false" to disable the validation checks.